node/v2/handler.js

```diff
@@ -289,6 +293,14 @@ TChannelV2Handler.prototype.handleCallResponse = function handleCallResponse(res
                 limit: '0x' + v2.CallResponse.MaxArg1Size.toString(16)
         }));
     }
+
+    var req = self.connection.ops.getOutReq(res.id);
+    self.connection.channel.inboundResponseSizeStat.increment(resFrame.size, {
+        'calling-service': !req ? null : req.headers.cn,
```

Comment:
yeah if its null it will be no-calling-service, no-service, etc. unknown-calling-service would probably be better though